### PR TITLE
Update create-clusters-topology.md on cross universe replication

### PR DESCRIPTION
**Pull request by mdbridge**
Currently there is a commented out section on this page that talks about xCluster replication using outdated terminology and gives incorrect information about consistency.

This diff changes just the commented out section (so no visible changes) with more recent terminology and somewhat better information about consistency.  Probably still needs additional editing if we were to make it visible.

Terminology changes:
  * sink -> target
  * cluster -> universe
    * we replicate between universes, not clusters
    * nonetheless, the feature is called xCluster replication :-(

I have temporarily uncommented out the section so you can see it in the preview.  I will return it to being commented out before I land it.

This page does not occur in stable so I will not be backporting it.

The edited version has a reference to part of the new xCluster architecture page so I will not be landing this until after landing https://github.com/yugabyte/yugabyte-db/pull/17938

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md [hunk 1]
```diff
@@ -158,71 +158,74 @@ To deploy a partition-by-region cluster, refer to [Partition by region](../creat
 - [Tablespaces](../../../explore/ysql-language-features/going-beyond-sql/tablespaces/)
 
 <!--
-## Cross-cluster
+-->
```
Comment (mdbridge) [inline docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md:161]: This is the line I will be removing before landing to comment out the section again

**Comment by mdbridge** [comment]
preview link to the section in question:
https://deploy-preview-18012--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-cloud/cloud-basics/create-clusters-topology/#cross-universe

File: docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md [hunk 2]
```diff
@@ -158,71 +158,74 @@ To deploy a partition-by-region cluster, refer to [Partition by region](../creat
...
+## Cross-universe
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md:162]:
```suggestion
## xCluster
```

File: docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md [hunk 3]
```diff
@@ -158,71 +158,74 @@ To deploy a partition-by-region cluster, refer to [Partition by region](../creat
...
 
 In situations where applications want to keep data in multiple clouds or in remote regions, YugabyteDB offers xCluster replication across two data centers or cloud regions. This can be either bi-directional in an active-active configuration, or uni-directional in an active-passive configuration.
 
 Here's how it works:
 
-1. You deploy two YugabyteDB clusters (typically) in different regions. Each cluster automatically replicates data in the cluster synchronously for strong consistency.
+1. You deploy two YugabyteDB universes (typically) in different regions. Each universe automatically replicates data in itself synchronously for strong consistency.
 
-2. You then set up cross cluster asynchronous replication from one cluster to another. This can be either bi-directional in active-active configurations, or uni-directional in active-passive configurations.
+2. You then set up cross universe asynchronous replication from one universe to another. This can be either bi-directional in active-active configurations, or uni-directional in active-passive configurations.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md:172]: cross cluster should be xCluster everywhere

**Review by ddhodge — APPROVED** [review]

**Review by lingamsandeep — APPROVED** [review]

File: docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md (hunk 2 above)
Comment (mdbridge) [reply docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md:161]: discussed off-line; no change

File: docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md (hunk 3 above)
Comment (mdbridge) [reply docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters-topology.md]: :+1:

**Pull request merged**